utils-plot.R

```diff
@@ -166,3 +166,11 @@ get_params <- function(geom_layer) {
 get_type <- function(object) {
     return(sub("^_", "", tolower(gsub("([A-Z])", "_\\1", class(object)[1]))))
 }
+
+round_df <- function(x) {
```

Comment:
This is not where this function belongs: it doesn't have anything to do with plots. Either create a new `utils-df.R` file, or just place it in `test.R`.

Reply:
I moved the file like you requested. The commit history got a little messy but im not sure how to fix it.